mines/trajectory[] agent_trajectory
